packages/backend/src/models/ValidationModel/ValidationModel.ts

```diff
@@ -114,6 +114,9 @@ export class ValidationModel {
                 `${UserTableName}.first_name`,
                 `${UserTableName}.last_name`,
                 `${SpaceTableName}.space_uuid`,
+                this.database.raw(
+                    `(SELECT ${AnalyticsChartViewsTableName}.timestamp FROM ${AnalyticsChartViewsTableName} WHERE saved_queries.saved_query_uuid = ${AnalyticsChartViewsTableName}.chart_uuid ORDER BY ${AnalyticsChartViewsTableName}.timestamp ASC LIMIT 1) as first_viewed_at`,
+                ),
```

Comment:
I think we have the wrong SQL here. This is for `first_viewed_at` instead of `views`

Reply:
oh yes, we do 😓 thanks for spotting this!